src/libraries/Native/Unix/System.Native/pal_autoreleasepool.c

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_autoreleasepool.h"
+#include <stdlib.h>
+
+void* SystemNative_CreateAutoreleasePool()
+{
+    return NULL;
```

Comment:
Let's assert in these two implementations. I don't think they should ever be called right?